2996_reflection/reflection.md

```diff
@@ -4328,13 +4334,17 @@ consteval void g(std::meta::info r, X<false> xv) {
 
   * [#.#]{.pnum} If the `$id-expression$` denotes an overload set `$S$`, overload resolution for the expression `&$S$` with no target shall select a unique function ([over.over]{.sref}); `$R$` represents that function.
 
+  * [#.#]{.pnum} Otherwise, if the `$id-expression$` denotes a variable declared by an `$init-capture$` ([expr.prim.lambda.capture]), `$R$` is ill-formed.
+
+  * [#.#]{.pnum} Otherwise, if the `$id-expression$` denotes a local entity `$E$` ([basic.pre]) for which there is an intervening lambda scope between the `$id-expression$` and the point at which that local entity was introduced, and a potentially-evaluated expression naming `$E$` within the innermost such scope would 
```

Comment:
This sentence just... stops.

Reply:
Lol WIP, didn't mean to push that 😂 Gotta keep you on your toes!